bukuserver/templates/bukuserver/bookmark_edit.html

```diff
@@ -23,7 +23,8 @@
   {{ super() }}
   {{ buku.limit_navigation_if_popup() }}
   {{ buku.script('bookmark.js') }}
+  <script>$('.submit-row').append($('.delete-form'))</script>
+  {{ buku.horizontal_form(excluding_popups=True) }}
   {{ buku.focus() }}
   {{ buku.link_saved() }}
-  <script>$('.submit-row').append($('.delete-form'))</script>
```

Comment:
Adding the Delete button was moved to before making the form horizontal (which in turn is placed before changing keyboard focus)